src/Makefile.am

```diff
@@ -480,7 +480,7 @@ endif
 
 check-formatting:
 	@echo "Checking source formatting style..."
-	(cd $(top_srcdir)/src; ../contrib/devtools/clang-format.py check clang-format-3.8 `cat ./.formatted-files`)
+	(cd $(top_srcdir)/src; ../contrib/devtools/clang-format.py check clang-format-3.8 `cat ./.gitattributes|cut -d ' ' -f 1`)
```

Comment:
This seems to be assuming that all files in .gitattributes shall be subjected to bitcoin-clang-format filter.

I would suggest filtering for those files explicitly when constructing the list to pass.